R/pkg/R/DataFrame.R

```diff
@@ -1203,7 +1203,7 @@ setMethod("collect",
               requireNamespace1 <- requireNamespace
               if (requireNamespace1("arrow", quietly = TRUE)) {
                 read_arrow <- get("read_arrow", envir = asNamespace("arrow"), inherits = FALSE)
-                as_tibble <- get("as_tibble", envir = asNamespace("arrow"))
+                as_tibble <- get0("as_tibble", envir = asNamespace("arrow"), ifnotfound = NULL)
```

Comment:
Sorry if I missed something but why is it `get0`?

Reply:
For 'get', if no object is found an error results. For `get0`,  'ifnotfound' returns if no object is found.

`as_tibble` could be not found on latest Arrow.

Reply:
oh, btw do you mind fixing here https://github.com/apache/spark/blob/master/R/pkg/R/deserialize.R#L240 too while we're here?

Reply:
oh, sure, no aware of another usage of it. Thanks.